src/Page-Components/AllArticles.jsx

```diff
@@ -0,0 +1,19 @@
+import ArticleCard from "../Components/Article-card"
+import '../css/Style.css'
+
+const AllArticles = ({articles}) => {
+    return (
```

Comment:
You should consider having some user feedback when this component renders that signifies it is still loading if we are waiting on the api request to come back.